File: qalog/src/main/java/com/gaumala/qalog/QA.java
package com.gaumala.qalog;

/**
 * Main API
 * This file is written in Java to make
 * the API more friendly to java projects.
 */
public class QA {
    protected static Logger logger = new Logger();

    public static void log(String line) {
        logger.log(System.currentTimeMillis(), line);
    }
}

File: qalog/src/main/java/com/gaumala/qalog/QA.kt
package com.gaumala.qalog

import java.io.File

object QA {
    internal val logger = Logger()

    fun log(entry: String) {
        logger.log(System.currentTimeMillis(), entry)
    }

    fun copyLog(dstFile: File) {
        logger.copy(dstFile.outputStream())
    }
}